demisto_sdk/commands/lint/resources/pylint_plugins/base_checker.py

```diff
@@ -122,6 +127,28 @@ def _quit_checker(self, node):
         except Exception:
             pass
 
+    def _execute_command_checker(self, node):
```

Comment:
please add a docstring and possibly some deev comments in the method

Reply:
done.